src/systemctl/systemctl-start-unit.c

```diff
@@ -381,10 +381,6 @@ int verb_start(int argc, char *argv[], void *userdata) {
         }
```

Comment:
I cannot get the commit message `Subscribe() is unnecessary if we RefUnit explicitly`.
Could you extend the commit message?